databricks_cli/unity_catalog/api.py

```diff
@@ -0,0 +1,236 @@
+# Databricks CLI
+# Copyright 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from databricks_cli.unity_catalog.uc_service import UnityCatalogService
+
+
+class UnityCatalogApi(object):
+    def __init__(self, api_client):
+        self.client = UnityCatalogService(api_client)
+
+    # Metastore APIs
+
+    def create_metastore(self, name, storage_root):
```

Comment:
not critical for now, but for consistency reasons, we should create one API wrapper class per entity, as it is on the backend. e.g. have `MetastoreApi`, `ExternalLocationApi` and etc. In couple of months this may get re-generated with the new process, so not sure if this comment should block the merge.

Reply:
+1 -- we can leave as is since this is not in the `sdk` package.